Build fails without `m2e.version` property set

When trying to perform a Maven build without the `m2e.version` property set, the `maven-artifact-notifier-core` module fails to build (`mvn package`) with:

    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:2.6:resources (default-resources) on project maven-artifact-notifier-core: Error loading property file '/user/path/artifact-listener/maven-artifact-notifier-core/${profile.filter.file}' -> [Help 1]

Expected results: the build should succeed out of the box without depending on Eclipse-specific setup.

--- a/maven-artifact-notifier-core/pom.xml
+++ b/maven-artifact-notifier-core/pom.xml
@@ -79,10 +79,7 @@
 		<profile>
 			<id>development</id>
 			<activation>
-				<property>
-					<!-- propriété présente en environnement m2eclipse -->
-					<name>m2e.version</name>
-				</property>
+				<activeByDefault>true</activeByDefault>
 			</activation>
 			<properties>
 				<profile.filter.file>src/main/filters/development.properties</profile.filter.file>

